ebmlite/core.py

```diff
@@ -1200,16 +1149,16 @@ def addElement(self, eid, ename, baseClass, attribs={}, parent=None,
         """
 
         def _getBool(d, k, default):
-            " Helper function to get a dictionary value cast to bool. "
+            """ Helper function to get a dictionary value cast to bool. """
             try:
-                return str(d[k]).strip()[0] in 'Tt1'
+                return bytearray(d[k]).strip()[0] in b'Tt1'
```

Comment:
Won't this fail if `d[k]` in some cases because there's no encoding specified?

Reply:
I actually just ran into this, and yes is does.  I put a fix in, but I'm not 100% happy with it.